oxbow/Cargo.toml

```diff
@@ -8,5 +8,6 @@ description = "Read specialized bioinformatic file formats as data frames in R,
 
 [dependencies]
 arrow = { workspace = true }
+bstr = { workspace = true }
```

Comment:
is this used anywhere?

Reply:
There's just one import of a trait bound in `model/tags.rs`. It can probably be worked around.

Reply:
Noodles provides some tag values as `BStr`. The `bstr::ByteSlice` trait lets you convert from those and from `&[u8]` directly to `&str`.